source/server-discovery-and-monitoring/tests/errors/write_errors_ignored.yml

```diff
@@ -0,0 +1,41 @@
+description: writeError field is ignored
+uri: mongodb://a/?replicaSet=rs
+phases:
+- description: Primary A is discovered
+  responses:
+  - - a:27017
+    - ok: 1
+      ismaster: true
+      hosts:
+      - a:27017
+      setName: rs
+      minWireVersion: 0
+      maxWireVersion: 9
+      topologyVersion: &topologyVersion_1_1
+        processId:
+          "$oid": '000000000000000000000001'
+        counter:
+          "$numberLong": '1'
+  outcome: &outcome
+    servers:
+      a:27017:
+        type: RSPrimary
+        setName: rs
+        topologyVersion: *topologyVersion_1_1
+        pool:
+          generation: 0
+    topologyType: ReplicaSetWithPrimary
+    logicalSessionTimeoutMinutes: null
+    setName: rs
+
+- description: Ignore command error with writeErrors field
+  applicationErrors:
+  - address: a:27017
+    when: afterHandshakeCompletes
+    maxWireVersion: 9
+    type: command
+    response:
+      ok: 1
+      writeErrors:
+      - { ok: 0, errmsg: NotMasterNoSlaveOk, code: 13435 }
```

Comment:
This shouldn't matter much, but I don't think individual documents in the `writeErrors` array have `ok` fields.